types/database/schema/ChannelList.types.ts

```diff
@@ -41,7 +41,8 @@ export type BetterSocialChannelType =
   | 'ANON_PM'
   | 'POST_NOTIFICATION'
   | 'ANON_POST_NOTIFICATION'
-  | 'TOPIC';
+  | 'TOPIC'
+  | 'ANON_TOPIC';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the 'ANON_TOPIC' type to the `BetterSocialChannelType` enum seems appropriate. It aligns with the PR's purpose of introducing anonymous topic chat functionality.

```diff
   | 'ANON_PM'
   | 'POST_NOTIFICATION'
   | 'ANON_POST_NOTIFICATION'
+  | 'TOPIC'
+  | 'ANON_TOPIC';
```
This change should not introduce any issues as long as all parts of the code that use this enum are updated to handle the new 'ANON_TOPIC' type correctly.


<!-- This is an auto-generated comment by OSS CodeRabbit -->